js/src/widgets/contextControls.js

```diff
@@ -69,11 +175,37 @@
                                    '<a class="mirador-osd-edit-mode hud-control" role="button" aria-label="Make a new annotation using mouse">',
                                    '<i class="fa fa-lg fa-edit"></i>',
                                    '</a>',
-                                   '{{/if}}',
-                                   '<a class="mirador-osd-refresh-mode hud-control" role="button" aria-label="Refresh annotations">',
+                                   '<a class="hud-control draw-tool" style="color:#abcdef;">',
+                                   '|',
+                                   '</a>',
+                                   '{{#each tools}}',
```

Comment:
There needs to be some error checking before calling the template.  If there are no available tools then it displays two horizontal bars next to each other, with nothing in between, and it is then impossible to create any sort of annotation because you can't select a shape to draw.  Perhaps, if ```availableAnnotationDrawingTools``` is empty, then we always default to at least rectangle?  @aeschylus, your input would be appreciated